lib/player.rb

```diff
@@ -0,0 +1,12 @@
+class Player
+  attr_accessor :player_name, :symbol, :position
```

Comment:
is `attr_accessor` necessary? or would `attr_reader` or `attr_writer` be more applicable?
See [screenshot(https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) of requirement as previously shown by @mikenath223